build.sh

```diff
@@ -1,15 +1,12 @@
 #!/bin/sh
 #
-# SPDX-License-Identifier: AGPL-3.0+
-#
```

Comment:
while I understand your reasoning, you can't just remove the license, it's kind of copyright infringement :p

To be fair, the blank line on line 2 shouldn't be there though, and the proper spdx identifier is `AGPL-3.0-or-later`

also, Hi Raymond!!

Reply:
Hey Oliver,

This repository actually doesn't have a license, so while the source is available, and it had that one identifier, it definately doesn't meet the standards for open source, certainly not for AGPL.
Thats why we are removing this specific identifier from all our repositories, it only adds confusion to what the code is actually licensed as.

Long time no seen Oliver!

Reply:
Hey Sanne! Didn't know you where still at Ultimaker :D

Anyway, this is not entirely true. How this generally works is, if there is no license information at all (not in the files, not in the root in a `LICENSE` or `COPYRIGHT` file, default copyright law applies, e.g. 'all rights reserved' by all authors. The copyright notice in the files helps you identify copyright holders, but is not definitive (there could be thousands of contributors that do not leave their copyright behind, VCS helps herein identifying those contributors however.

So there is no 'generic' license set for this repository, this is true, however, there's also the copyright notice (SPDX license identifier) at the top of each file. This takes precedence over the generic `LICENSE` file. E.g. your `LICENSE` file could state 'MIT' but all your files say 'GPL', Well, then all non-marked files are indeed MIT licensed, but every file that mentions GPL, is GPL licensed. This is how you often get those notices 'This project is X Licensed and some files are Y licensed' or similar.

Now, if there is no generic license, but most/all of the headers state a certain license, you have to assume, that this was the intend, and the central license file is simple missing and thus the license from the headers was implied to be applicable to the repository as a whole. But indeed, this can be a little dicey at times. E.g. a repo with thousands of files, but only one contains a header, does that mean all files carry that license? At this point you are looking at intend, and the virality of the license. In case of doubt however, standard copyright applies and you'd have to contact the original author to specify the license.

I'm not sure what you mean in terms of 'it does not meet standard of open source, and certainly not for AGPL', but this simply not true. Sloppy maybe, it is better and nicer to add the default copyright notice in the root of the project, but there is no hard requirement. Also, there is no difference between the AGPL or many of the other licenses.

Finally, there really is no confusion for files that have this license. It is VERY explicit. This file, without a doubt, is licensed as `AGPL-3.0-or-later. So by removing this notice, and not having a `LICENSE` you now either rely on the fact that other files carry this marking or worse, you remove it and potentially move it into the default copyright domain, or worse a different license. By doing so, you are violating copyright and the license.

Ironically, the 'standard' in opensource, is to remove any duplicity or uncertainty, by ensuring there exists a copyright header on all files. Many projects in retrospect wrote scripts to add the SDPX license identifier to files, because it is now just a single line. So you are doing the opposite of what is 'standards for open source' and to be honest, it's a little shady too :(